posts/2025-12-02-project-goals-nov-update.md

```diff
@@ -0,0 +1,238 @@
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/121'><strong>Stabilize parallel front end</strong></a></div>
+    <div style="flex: initial;"><progress value="1" max="3"></progress>
+</div>
+</div>
+
+* Still in the process of determining the cause of the deadlock through local testing and compiler code analysis.
+* **Help wanted:** Help test the deadlock code in the [issue list](https://github.com/rust-lang/rust/labels/WG-compiler-parallel) and try to reproduce the issue.
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/126'><strong>Survey tools suitability for Std safety verification</strong></a></div>
+    <div style="flex: initial;"><progress value="3" max="6"></progress>
+</div>
+</div>
+
+* A [new partnership](https://foundation.rust-lang.org/news/rust-foundation-collaborates-with-aws-initiative-to-verify-rust-standard-libraries/) between the Rust Foundation and AWS will help fund this effort. The [verification challenges](https://model-checking.github.io/verify-rust-std/challenges.html) in the [verify-rust-std fork](https://github.com/model-checking/verify-rust-std) now have financial rewards for those completing them.
+* **Help wanted:** Help needed to write more contracts, to integrate new tools, to review [pull requests](https://github.com/model-checking/verify-rust-std/pulls) or to participate in the [repository discussions](https://github.com/model-checking/verify-rust-std/discussions).
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/122'><strong>Testing infra + contributors for a-mir-formality</strong></a></div>
+    <div style="flex: initial;"><img src="https://img.shields.io/badge/Status-Will%20not%20complete%20%3A%28-yellow" alt="Will not complete"></img>
+</div>
+</div>
+
+* We decided to close this goal as we have not been making steady progress. We are evaluating what to propose the 2025h1 round of goals.
+
+## Goals without updates
+
+The following goals have not received updates in the last month:
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/103'><strong>Associated type position impl trait</strong></a></div>
+    <div style="flex: initial;"><img src="https://img.shields.io/badge/Status-Will%20not%20complete%20%3A%28-yellow" alt="Will not complete"></img>
+</div>
+</div><div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/111'><strong>Implement &quot;merged doctests&quot; to save doctest time</strong></a></div>
+    <div style="flex: initial;"><img src="https://img.shields.io/badge/Status-Completed%20%3D%29-green" alt="Completed"></img>
+</div>
```

Comment:
Some of the goals listed as "goals which have not received updates" are not going to receive any 😅:
- this rustdoc goal has already been completed in August
- the "Associated type position impl trait" just above was also closed in October

Reply:
Yeah, I figured that was ok.